client/webserver/site/src/js/forms.ts

```diff
@@ -1092,7 +1090,7 @@ export class FeeAssetSelectionForm {
   }
 
   refresh () {
-    this.setExchange(app().exchanges[this.host])
+    this.setExchange(this.xc)
   }
```

Comment:
Information about this exchange might have changed in `app().exchanges` I think.